File: android/app/src/main/java/com/agenthub/app/data/model/Session.kt
package com.agenthub.app.data.model

data class Session(
    val id: String,
    val title: String = "",
    val createdAt: Long = System.currentTimeMillis(),
    val updatedAt: Long = System.currentTimeMillis(),
    val isPinned: Boolean = false,
    val messageCount: Int = 0,
    val summary: String = ""
)

File: ios/AgentControlCenter/Models/Session.swift
import Foundation

// MARK: - Session Model
// 对应 protocol/schemas/session-schema.json

/// 会话模型。一个 Session 代表一次连续对话，包含多条 Message。
struct Session: Codable, Identifiable, Equatable {
    var id: String
    var title: String
    var createdAt: Int64
    var updatedAt: Int64
    var isPinned: Bool = false
    var messageCount: Int = 0
    var summary: String = ""
}
